### PR TITLE
Credit card fraud dataset

### PR DESCRIPTION
**Pull request by jjbrosnan**
Contains all necessary information and citations.

### CODE REVIEW

**Comment by jjbrosnan** [comment]
I have read the CLA Document and I hereby sign the CLA

**Review by jakemulf — DISMISSED** [review]

File: CreditCardFraud/README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Credit Card Fraud Data
+
+This directory contains a [credit card purchase dataset](https://www.kaggle.com/mlg-ulb/creditcardfraud).  The data set contains anonymized parameters for 48 hours worth of purchases by European cardholders.  There are 284,807 purchases in total, and 492 of them are fraudulent.
```
Comment (margaretkennedy) [inline CreditCardFraud/README.md:3]:
```suggestion
This directory contains a [credit card purchase dataset](https://www.kaggle.com/mlg-ulb/creditcardfraud) with anonymized parameters for 48 hours worth of purchases by European cardholders.  There are 284,807 purchases in total, and 492 of them are fraudulent.
```

File: CreditCardFraud/README.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# Credit Card Fraud Data
```
Comment (margaretkennedy) [inline CreditCardFraud/README.md:1]:
```suggestion
# Credit card fraud data
```

File: CreditCardFraud/README.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+This directory contains a [credit card purchase dataset](https://www.kaggle.com/mlg-ulb/creditcardfraud).  The data set contains anonymized parameters for 48 hours worth of purchases by European cardholders.  There are 284,807 purchases in total, and 492 of them are fraudulent.
+
+## Fields in `creditcard.csv`
+
+The `creditcard.csv` file contains 31 columns of data.  These columns are:
+
+- **Time** - The integer number of seconds after the first purchase in the file.
+- **V1 - V28** - The 28 anonymized PCA transformed purchase parameters.
+- **Amount** - The purchase amount in Euros.
+- **Class** - A flag indicating a valid (0) or fraudulent (1) purchase
```
Comment (margaretkennedy) [inline CreditCardFraud/README.md:12]:
```suggestion
- **Class** - A flag indicating a valid (0) or fraudulent (1) purchase.
```

File: CreditCardFraud/README.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+# Source and license
+
+This data is taken directly from [Kaggle](https://kaggle.com).  It is provided here for demonstrative use without any warranty as to the accuracy, reliability, or completeness of the data.  Credit for the dataset goes to:
```
Comment (margaretkennedy) [inline CreditCardFraud/README.md:16]:
```suggestion
This data is taken directly from [Kaggle](https://kaggle.com).  It is provided here for demonstrative use without any warranty as to the accuracy, reliability, or completeness of the data.  

Credit for the dataset goes to:
```

Comment (margaretkennedy) [reply CreditCardFraud/README.md:16]: This looks more like a Works Cited List. How about, "The following sources contributed to this dataset:"

File: CreditCardFraud/README.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+
+Andrea Dal Pozzolo, Olivier Caelen, Reid A. Johnson and Gianluca Bontempi. [Calibrating Probability with Undersampling for Unbalanced Classification](https://www.researchgate.net/publication/283349138_Calibrating_Probability_with_Undersampling_for_Unbalanced_Classification). In Symposium on Computational Intelligence and Data Mining (CIDM), IEEE, 2015
```
Comment (margaretkennedy) [inline CreditCardFraud/README.md:18]:
```suggestion
Dal Pozzolo, Andrea,  Olivier Caelen, Reid A. Johnson and Gianluca Bontempi. [Calibrating Probability with Undersampling for Unbalanced Classification](https://www.researchgate.net/publication/283349138_Calibrating_Probability_with_Undersampling_for_Unbalanced_Classification). In Symposium on Computational Intelligence and Data Mining (CIDM), IEEE, 2015
```

File: CreditCardFraud/README.md [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+
+Dal Pozzolo, Andrea; Boracchi, Giacomo; Caelen, Olivier; Alippi, Cesare; Bontempi, Gianluca. [Credit card fraud detection: a realistic modeling and a novel learning strategy](https://www.researchgate.net/publication/319867396_Credit_Card_Fraud_Detection_A_Realistic_Modeling_and_a_Novel_Learning_Strategy), IEEE transactions on neural networks and learning systems,29,8,3784-3797,2018,IEEE
+
+Dal Pozzolo, Andrea [Adaptive Machine learning for credit card fraud detection](http://di.ulb.ac.be/map/adalpozz/pdf/Dalpozzolo2015PhD.pdf) ULB MLG PhD thesis (supervised by G. Bontempi)
+
+Carcillo, Fabrizio; Dal Pozzolo, Andrea; Le Borgne, Yann-Aël; Caelen, Olivier; Mazzer, Yannis; Bontempi, Gianluca. [Scarff: a scalable framework for streaming credit card fraud detection with Spark](https://www.researchgate.net/publication/319616537_SCARFF_a_Scalable_Framework_for_Streaming_Credit_Card_Fraud_Detection_with_Spark), Information fusion,41, 182-194,2018,Elsevier
```
Comment (margaretkennedy) [inline CreditCardFraud/README.md:26]: can we alphabetize this? Bertrand first

**Review by margaretkennedy — CHANGES_REQUESTED** [review]

File: CreditCardFraud/README.md [hunk 7]
```diff
@@ -0,0 +1,36 @@
+# Credit card fraud data
+
+This directory contains a [credit card purchase dataset](https://www.kaggle.com/mlg-ulb/creditcardfraud) with anonymized parameters for 48 hours worth of purchases by European cardholders.  There are 284,807 purchases in total, and 492 of them are fraudulent.
```
Comment (chipkent) [inline CreditCardFraud/README.md:3]:
```suggestion
This directory contains a [credit card purchase dataset](https://www.kaggle.com/mlg-ulb/creditcardfraud) with anonymized parameters.  48 hours of purchases by European cardholders are recorded.  There are 284,807 purchases in total, and 492 of them are fraudulent.
```

File: CreditCardFraud/README.md [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+The `creditcard.csv` file contains 31 columns of data.  These columns are:
+
+- **Time** - The integer number of seconds after the first purchase in the file.
+- **V1 - V28** - The 28 anonymized PCA transformed purchase parameters.
+- **Amount** - The purchase amount in Euros.
+- **Class** - A flag indicating a valid (0) or fraudulent (1) purchase.
+
+# Source and license
+
+This data is taken directly from [Kaggle](https://kaggle.com).  It is provided here for demonstrative use without any warranty as to the accuracy, reliability, or completeness of the data.
```
Comment (chipkent) [inline CreditCardFraud/README.md:16]:
```suggestion
This data was obtained from [Kaggle](https://kaggle.com).  It is provided here for demonstrative use without any warranty as to the accuracy, reliability, or completeness of the data.  
```

File: README.md [hunk 9]
```diff
@@ -8,6 +8,7 @@ This repository contains open source data sets.  They are intended to be used as
 
 The following folders can be found in this repository:
 
+- **[`CreditCardFraud`](./CreditCardFraud)** - Anonymized credit card purchase data including fraudulent purchases.
```
Comment (chipkent) [inline README.md:11]:
```suggestion
- **[`CreditCardFraud`](./CreditCardFraud)** - Anonymized credit card purchase data, including fraudulent purchases.
```

**Review by chipkent — CHANGES_REQUESTED** [review]

**Review by chipkent — APPROVED** [review]

**Pull request merged**